Admin/ManagerInterface.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace LAG\AdminBundle\Admin;
+
+use Doctrine\ORM\EntityRepository;
+
+interface ManagerInterface
```

Comment:
Could all the manager thing be in the base bundle or even better, live in its own component/bundle?